src/NUnitFramework/framework/Internal/Commands/OneTimeSetUpCommand.cs

```diff
@@ -65,15 +65,8 @@ public override TestResult Execute(TestExecutionContext context)
             if (_typeInfo != null)
             {
                 // Use pre-constructed fixture if available, otherwise construct it
-                if (!_typeInfo.IsStaticClass)
-                {
-                    context.TestObject = _suite.Fixture ?? _typeInfo.Construct(_arguments);
-                    if (_suite.Fixture == null)
-                    {
-                        _suite.Fixture = context.TestObject;
-                    }
-                    Test.Fixture = _suite.Fixture;
-                }
+                if (!_typeInfo.IsStaticClass && context.TestObject == null)
+                    context.TestObject = _typeInfo.Construct(_arguments);
```

Comment:
Yes, I believe so. Sorry - could have used a comment! I wasn't much of a fan of doing that however, it seems like we could use a proper mechanism for 'seeding' testobjects? 

Just to clarify - where are testobjects intended to be created - I presume this should happen in the OneTimeSetUp for the testfixture, and then be passed context to context for each tests?